scripts/bash/run_enceladus.sh

```diff
@@ -440,6 +447,9 @@ add_to_cmd_line "--experimental-mapping-rule" ${EXPERIMENTAL_MAPPING_RULE}
 add_to_cmd_line "--catalyst-workaround" ${CATALYST_WORKAROUND}
 add_to_cmd_line "--autoclean-std-folder" ${AUTOCLEAN_STD_FOLDER}
 add_to_cmd_line "--persist-storage-level" ${PERSIST_STORAGE_LEVEL}
+if [ "$HELP_CALL" == "1" ]; then
```

Comment:
I don't like this personally, but I can understand there is no real compelling reason to do it otherwise as it copies what jars do. But here are my findings:

If I run something with an issue in the preceding arguments, there will be errors and then help, which is weird on its own. But also, not all errors are printed.

If I run `...--dataset-version alfa --raw-format parquet --trimValues true --help...` I get an error for `dataset-version` and help, but I don't get that raw-format parquet and trimValues do not work together.

**What I get**
```
Error: Option --dataset-version expects a number but was given 'alfa'
Enceladus v2.15.0-SNAPSHOT
Usage: Standardization Job [options]
...
```
**What I expect (option a)**
just help. Nothing else. Ignore arguments

**What I expect (option b)**
```
Error: Option --dataset-version expects a number but was given 'alfa'
Error: The --trimValues option is supported only for FixedWidth format
Enceladus v2.15.0-SNAPSHOT
Usage: Standardization Job [options]
...
```

I am aware the jar behaves the same. But if we are providing **helper** scripts I would probably have them be more "precise" (cannot come up with the word I am looking for).